build/scripts/build-volplugin-containers.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+docker build -t contiv/volplugin .
+
+# Ensure that docker is running with MountFlags=shared
+sudo sed -i 's/MountFlags=slave/MountFlags=shared/g' /usr/lib/systemd/system/docker.service
+sleep 1
```

Comment:
Is this 1 second sleep necessary?  `systemctl daemon-reload` won't run before `sed` is finished, so I don't think there's any possible race condition here...